docs/par2cron_bundle_unpack.md

```
@@ -0,0 +1,49 @@
+By default bundles will only unpack when it can be guaranteed
+that all original files can be replicated and are not corrupted.
+In some cases it may be needed to unpack corrupted bundles, this
+is achievable (as a last resort) by using the --force argument.
+
+To exclude directories from this operation, put ignore files:
+  - ".par2cron-ignore" (ignore directory)
+  - ".par2cron-ignore-all" (ignore directory and subdirectories)
+
+Full documentation at: https://github.com/desertwitch/par2cron
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)